nixos/doc/manual/release-notes/rl-1909.xml

```diff
@@ -591,6 +591,20 @@
      The defaults from fontconfig are sufficient.
     </para>
    </listitem>
+   <listitem>
+    <para>
+      The <literal>crashplan</literal> package and the
+      <literal>crashplan</literal> service have been removed from nixpkgs due to
+      crashplan shutting down the service.
+    </para>
+   </listitem>
+   <listitem>
```

Comment:
Duplicate entry

Reply:
They are actually two packages and two services: 
* The `crashplan` package and `crashplan` service for home users, discontinued by crashplan.  
* The `crashplansb` package and `crashplan-small-business` service, currently offered by crashplan but unmaintained and broken in nixpkgs.    

I can merge both notes in a single one if that is preferred.

Reply:
SGTM.